vitrina/datasets/templates/vitrina/datasets/filters.html

```diff
@@ -1,9 +1,18 @@
 {% load i18n %}
 
 <nav class="category-panel">
-    <p class="category-panel-heading">
-        <strong>{% translate "Rinkinio būsena" as datasetStatus %}{{ datasetStatus|upper}}</strong>
-    </p>
+    {% if labels %}
+        <a href="{% url 'dataset-list' %}?{{ request.GET.urlencode }}">
+            <p class="category-panel-heading">
+                <strong><- {% translate "Atgal į rinkinių sąrašą" %}{{ datasetStatus|upper}}</strong>
```

Comment:
```suggestion
                <strong>&lt;- {% translate "Atgal į rinkinių sąrašą" %}{{ datasetStatus|upper}}</strong>
```

Always escape spatial html characters.